db/schema.rb

```diff
@@ -57,6 +57,10 @@
     t.boolean "published", default: false, null: false
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.boolean "use_email", default: false
```

Comment:
Why do both events and users have these use methods? Does an event need to know about how its being sent to the users?